internal_testsuites/golang/scripts/test.sh

```diff
@@ -17,8 +17,12 @@ KUBERNETES_TIMEOUT="510s" # K8S takes longer than docker
 TESTSUITE_CLUSTER_BACKEND_DOCKER="docker"
```

Comment:
With this change we can potentailly take `KUBERNETES_TIMEOUT` back to `8ms`? Or maybe same time as docker even?

Reply:
With the run_sh changes; we can reduce this definitely! Will bring it back down :-)